assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+class Person
+{
```

Comment:
This curly brace needs to be on line 2. You need to break your C++ styling conventions, and embrace the JS conventions.